package.json

```diff
@@ -1,12 +1,6 @@
 {
   "name": "secure-handlebars",
   "version": "1.1.1",
-  "licenses": [
-    {
-      "type": "BSD",
-      "url": "https://github.com/yahoo/secure-handlebars/blob/master/LICENSE"
-    }
-  ],
```

Comment:
this is deprecated as illustrated in https://docs.npmjs.com/files/package.json#license